frontend/src/components/dashboard/FileName.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import PhotoUploadIcon from '@material-ui/icons/AddPhotoAlternate';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = theme => ({
+  root: {
+    '& .MuiTextField-root': {
+      marginLeft: 0,
+      width: '150ch',
+    },
+    height: '40px',
+    margin: theme.spacing(1),
+    width: '75ch',
+  },
+  input: {
+    display: 'none',
+  },
+});
+
+class FileName extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+       fileName: '',
+    };
+  }
+
+  onChange = e => {
+
+    switch (e.target.name) {
+      case 'selectedFile':
+        if(e.target.files.length > 0) {
```

Comment:
Might be worth to use the state hook instead so we can stay consistent with MUI style 
https://reactjs.org/docs/hooks-state.html